ga4gh/frontend.py

```diff
@@ -384,8 +403,139 @@ def searchDatasets(version):
         version, flask.request, app.backend.searchDatasets)
 
 
+# The below paths have not yet been implemented
+
+
+@app.route('/<version>/callsets/<no(search):id>')
```

Comment:
I don't think it's terribly unusual to have certain reserved paths in REST (in any event, it's defined by the schema, so our hands are tied somewhat).  Alternatives would be (if `foo` is the resource):

- have an attribute in the POST payload to `/foo` specifying the action (this seems like overloading paths = bad)
- have `/foo` for CRUD methods and `/foos` for more advanced ones over all the `foo` objects like search (but foo vs foos distinction could be seen as confusing)... that is, if we even ever implement CRUD methods
- organize the URL space differently and have a `/search/foos` endpoint (seems to break the object-then-action URL simplicity)
- have some other piece of data be sent along with the usual payload to `/foo`, like a header or something (confusing)
- just assume a POST to `/foo` means we want to search (possible, but would make any CRUD implementation backwards-incompatible)

None of these seem as attractive as what we're doing now... in any event, `search` seems like a pretty improbable ID (although, I don't doubt that someone, somewhere...)